assets/case/edit.js

```diff
@@ -1,4 +1,13 @@
 /* global $ */
+
+// On load trigger change once to preselect municipality
+window.addEventListener('load', function () {
+  const $identificationTypes = $("[id*='Identification_type']")
+  $identificationTypes.each(function () {
+    $(this).trigger('change')
+  })
+})
+
```

Comment:
You could also chain this when setting up the `click` handler, i.e.

```js
$(this)
  …
  .on('change', …)
  .trigger('change')
```

For consistency you should also to this in `new.js`.